### DISCUSSION TITLE
Outdated roadmap page

### DISCUSSION THREAD

**Discussion by Tony20221** [OrchardCMS/OrchardCore#14175] [General]
The last edit on the [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap) page was two years ago. And I see a phrase like 'The software is finished'. What does this all mean? What we have of Orchard Core is all that we will ever get?
Don't expect any major new features?

**Comment by Piedone** [comment]
The intent there I think was that Orchard is ready for production use now, not that it won't get updates (which it obviously got a lot since then), see "and by finished, we mean there are no show-stopping, little-children-killing bugs in it. That we know of."

But yeah, we should update that page.

**Reply by MikeAlhayek** [reply]
I think the Roadmap should also be moved out of Wiki and into the docs website.
